jsonschema-use-cases.xml

```diff
@@ -0,0 +1,300 @@
+                <t>
+                    Application developers may wish to express a wide variety of constraints that would be impractical to require every validator to implement.
+                </t>
+                <t>
+                    Like in the <xref target="partial-validation">"Partial validation"</xref> use case, validators need to understand when the extension is not supported, and allow the application to determine if validation may continue without it. For example, if a Web API does not use XML, it would be unnecessary for its JSON validator to support keywords dealing with XML, so in this case, it's appropriate for its JSON validator to reject constraints dealing with XML altogether.
+                </t>
+            </section>
+        </section>
+
+        <section title="Use cases for extensions">
+            <t>
+                This section lists use cases that not required for interoperability in media type handling, but which should be defined as as standard extensions to maximize support and feature interoperability.
+                As specified in <xref target="extension-points">"Extension points,"</xref>
+                validators should gracefully degrade in the presence of extensions that they do not support.
+            </t>
+
+            <section title="External validation">
+                <t>
+                    JSON may embed resources of other media types, like base64 or hex-encoded documents; and may wish to pass off validation of these documents to another software tool.
+                </t>
+            </section>
+
+            <section title="Intra-document data consistency validation">
+                <t>
+                    A JSON document may carry relational data whose consistency must be verified. Example constraints include:
+                </t>
+                <ul>
+                    <li>One-to-one calculations, like that children's birthdates come after their parent's birthdates;</li>
+                    <li>One-to-many calculations, like a reference to an anchor points to an anchor defined somewhere in the same document.</li>
+                </ul>
+            </section>
+
+            <section title="Inter-database consistency validation">
+                <t>
+                    A JSON document may carry relational data whose consistency must be verified. Example constraints include:
+                </t>
+                <ul>
+                    <li>References to a user ID points to a user in a central users database.</li>
+                    <li>A supplied email address has been verified by the user.</li>
+                </ul>
+            </section>
+
+        </section>
+
+        <section title="Requirements" anchor="requirements">
+            <t>
+                Requirement 1.
+            </t>
+            <section title="R1. " anchor="r1">
+                <t>
+                    Requirement 1.
```

Comment:
Missed description?

Reply:
Yes, this whole "Requirements" section needs to be filled out. Or maybe removed.